src/identity/apis/org.ts

```diff
@@ -117,6 +118,19 @@ export const fetchDefaultAccountDefaultOrg = async (): Promise<
   }
 }
 
+// fetch data regarding whether the user can create new orgs, and associated upgrade options.
+export const fetchOrgCreationAllowance = async () => {
+  const response = await getAllowancesOrgsCreate({})
+
+  if (response.status !== 200) {
+    throw new GenericError(
+      'Failed to determine whether this user can create a new organization.'
+    )
+  }
+
+  return response.data
+}
+
```

Comment:
Calling this an `allowance` based on some discussion with Randy about the right nomenclature. Calling it a 'permission' is a little misleading because it's not a roles-and-permissions issue, nor is it really an entitlement. "Am I allowed to do X" is a simpler and easier way to characterize this.

Note that what it returns is an object with two `boolean`s - `allowed` and `upgradesAvailable`.